app-admin/cdist/Manifest

```diff
@@ -1 +1 @@
-DIST cdist-7.0.0.tar.gz 1395298 BLAKE2B 0dc11302b4a63e24f31d598df63dc5da784cb01d57ade9b67da277b7c27f66b4c31891ce710c3acdc100e92500b0e80e22cf01882b1669fa714d70c44879f7f0 SHA512 bd87b52603443bc9ffd81785a73bc07387e0555da34b04f97619beb39a1bfcf488b84a6a37fac4fe49b9b002080c5f20a50b3f08ac3665c6c2143a4aa4ee47b7
+DIST cdist-7.0.0.tar.gz 1397360 BLAKE2B 52975321d75a39677a1b299fce84c0e194bf6a11925492862ed10c108367b617d67cce035cc89291e54119ebfcb32e87f81f3b3c5ee19e249a7310303016ea87 SHA512 88c6f7ff68c42dc7db37a5df1fa48ee06b1dc505ec78148bc65923dab68c9dbcf302dc2fb7197f43202ffbcb11c3a82b7303398a805b445f18e43b6fec30c813
```

Comment:
This is simply broken because the mirror network will have the old version. Also people with cached distfiles will get checksum failures but at least they could delete it and start over if they can be guaranteed the server will return the new checksum.

Reply:
damn - forgot that, i have to rename the package...